src/service/calculator/calculator_report.py

```diff
@@ -72,6 +72,27 @@ def calculate_metrics(self, company: dict) -> None:
         )
         growth_range = self.get_metric_range(growth, "growth profile")
         company["margin_group"] = growth_range
+        company["gross_profit"] = self.calculator.calculate_gross_profit(
+            revenue, company.get("actuals_cost_of_goods")
+        )
+        company["gross_margin"] = self.calculator.calculate_gross_margin(
+            revenue, company.get("actuals_cost_of_goods")
+        )
+        company[
+            "sales_and_marketing"
+        ] = self.calculator.calculate_sales_and_marketing_of_revenue(
+            revenue, company.get("actuals_sales_of_marketing")
```

Comment:
You have different names for the same metric in report and repository, please change that.
The name should be the same.